### DISCUSSION TITLE
[Scrum Prep] Milestone 1 - Week 1

### DISCUSSION THREAD

**Discussion by cristianramos9** [uprm-inso4101-2024-2025-s1/semester-project-regiupr#16] [Scrum Preps]
Write your scrum prep for  week 1 of milestone 1
Week: August 30 - September 5

**Comment by cristianramos9** [comment]
No work on tasks.
Have been doing research on databases, as discussed on the team meeting.

**Comment by nelson-escalona** [comment]
Worked on database design.

**Comment by carlospepin23** [comment]
During the week, I researched how to implement the database both locally and on the AWS web service. In addition, I worked on the database schema that will be implemented.

**Comment by J3SSY-ANDU** [comment]
I've been focused on researching different database options for our project to ensure we choose the most suitable one.

**Comment by Gian-Go** [comment]
No work done on Application Screens yet. I have been getting familiar with Python, Tkinter, and Paramiko, though I have not been able to dedicate much time to this during this week.

**Comment by nicolle-feliciano** [comment]
Did research on Handlers and DAOs as suggested by the manager.

**Comment by PedroRodz** [comment]
Attended team meeting on database design and discussed database schema

**Comment by Yamil-Serrano** [comment]
We had not yet worked on any tasks, we were planning how to organize ourselves and how we could create an intuitive code to facilitate data management.

**Comment by christianmedina-dev** [comment]
Application Screens team, this week I've been researching and familiarizing myself more with Paramiko, Tkinter and PyQT5 to better accomplish our tasks.

**Comment by cristianMarcial** [comment]
working on task #6
